rubin_sim/maf/plots/plot_handler.py

```diff
@@ -602,11 +602,11 @@ def plot(
         # Make plot.
         fignum = None
         for m_b, plot_dict in zip(self.m_bundles, self.plot_dicts):
-            if m_b.metric_values is None:
+            if (m_b.metric_values is None) | (
+                np.size(np.where(~m_b.metric_values.mask)) == 0
+            ):
                 # Skip this metricBundle.
-                msg = 'MetricBundle (%s) has no attribute "metric_values".' % (
-                    m_b.file_root
-                )
+                msg = 'MetricBundle (%s) has no "metric_values".' % (m_b.file_root)
```

Comment:
I think I lost my previous comment so I'll write it again .. 
the metric_values being None comes from a different source than the values all being masked, and sending different error messages is helpful for the user when figuring out what went wrong.